include/ccf/ds/logger.h

```diff
@@ -218,23 +181,8 @@ namespace ccf::logger
 
     preamble += file_line_data;
 
-    if (enclave_offset.has_value())
-    {
-      // Sample: "2019-07-19 18:53:25.690183 -0.130 " where -0.130 indicates
-      // that the time inside the enclave was 130 milliseconds earlier than
-      // the host timestamp printed on the line
-      return fmt::format(
-        "{} {:+01.3f} {:<3} {:<45}| {}\n",
-        get_timestamp(host_tm, host_ts),
-        enclave_offset.value(),
-        ll.thread_id,
-        preamble,
-        ll.msg);
-    }
-    // Padding on the right to align the rest of the message
-    // with lines that contain enclave time offsets
     return fmt::format(
-      "{}        {:<3} {:<45}| {}\n",
+      "{} {:<3} {:<45}| {}\n",
```

Comment:
Accidental benefit - shorter log lines! We don't need to align with lines containing the offset, so we remove some padding whitespace.